teamcity: failed tests on master: check/TestStyle/TestReturnCheck

The following tests appear to have failed:

[#40846](https://teamcity.cockroachdb.com/viewLog.html?buildId=40846):

```
--- FAIL: check/TestStyle/TestReturnCheck (24.937s)
null

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/storage/replica_test.go
+++ b/pkg/storage/replica_test.go
@@ -6258,7 +6258,7 @@ func TestReplicaTimestampCacheBumpNotLost(t *testing.T) {
 	ctx := tc.store.AnnotateCtx(context.TODO())
 	key := keys.LocalMax
 
-	txn := newTransaction("test", key, 1, enginepb.SERIALIZABLE, tc.clock)
+	txn := newTransaction("test", key, 1, enginepb.SERIALIZABLE, tc.Clock())
 	origTxn := txn.Clone()
 
 	minNewTS := func() hlc.Timestamp {

